src/main/java/frc/robot/subsystems/intake/IntakeConstants.java

```diff
@@ -0,0 +1,62 @@
+package frc.robot.subsystems.intake;
+
+public class IntakeConstants {
+
+  private static final String CONSTRUCTOR_EXCEPTION = "constant class";
+
+  private IntakeConstants() {
+    throw new IllegalStateException(CONSTRUCTOR_EXCEPTION);
+  }
+
+  public static final int INTAKE_ROLLER_MOTOR_ID = 18;
+  public static final int INTAKE_KICKER_MOTOR_ID = 21;
+
+  public static final int INTAKE_RIGHT_ROLLER_IR_SENSOR_ID = 0;
+  public static final int INTAKE_KICKER_IR_SENSOR_ID = 1;
+  public static final int INTAKE_SHOOTER_IR_SENSOR_ID = 2;
+
+  // add all current limiting
+
+  public static final double INTAKE_ROLLER_MOTORS_KP = 10.0;
+  public static final double INTAKE_ROLLER_MOTORS_KI = 0.0;
+  public static final double INTAKE_ROLLER_MOTORS_KD = 0.0;
+  public static final double INTAKE_ROLLER_MOTORS_KS = 6.5;
+
+  public static final double INTAKE_DRUM_MOTOR_KP = 1.0;
+  public static final double INTAKE_DRUM_MOTOR_KI = 0.0;
+  public static final double INTAKE_DRUM_MOTOR_KD = 0.0;
+  public static final double INTAKE_DRUM_MOTOR_KS = 4.0;
+
+  public static final double INTAKE_KICKER_MOTOR_KP = 1.0;
+  public static final double INTAKE_KICKER_MOTOR_KI = 0.0;
+  public static final double INTAKE_KICKER_MOTOR_KD = 0.0;
+  public static final double INTAKE_KICKER_MOTOR_KS = 4.0;
+
+  // velocities
+
+  public static final double INTAKE_VELOCITY_ROLLERS_RPS = 40.0;
+  public static final double REPEL_VELOCITY_ROLLERS_RPS = -40.0;
+  public static final double KICKER_VELOCITY_RPS = 6.0;
+
+  public static final double ROLLER_VELOCITY_TOLERANCE = 1.0;
+  public static final double KICKER_VELOCITY_TOLERANCE = 0.5;
+
+  // current limits
+
+  public static final double ROLLERS_CONTINUOUS_CURRENT_LIMIT = 15.0;
```

Comment:
These are supply current limits, which is correct. Let's change the name to distinguish supply current limits from stator current limits. Let's also add stator current limits with a continuous of 25 and a peak of 30. We can talk about this.

Reply:
We want stator current limits to protect the Falcon 500 if the intake or kicker jams. (We use supply current limits to manage our current budget and prevent brownouts.)